.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
+  - '6'
```

Comment:
`stable` is no longer a thing. It's `node`, which refers to the latest, but I don't see the point in testing on Node.js 7. We can add Node.js 8 in a month when it's out.